scripts/install

```diff
@@ -38,76 +42,76 @@ grep -q -R 'jessie-backports' /etc/apt/sources.list{,.d} || {
 }
 
 # Install dependencies
-ynh_package_install_from_equivs ../conf/${DEPS_PKG_NAME}.control \
-  || ynh_die "Unable to install dependencies"
+ynh_package_install_from_equivs ../conf/${DEPS_PKG_NAME}.control
```

Comment:
You need to put all dependencies (i.e. packages that are required for the app to work). apt will make its job figuring out which ones are already there, and which ones are to be de-installed when removing this meta-package.